(doc) Add documentation for Commercial Validation Rule (#293)
chocolatey/docs: input/en-us/community-repository/moderation/package-validator/rules/cpmr0071.md

---
Order: 0071
xref: cpmr0071
Title: CPMR0071 - Script uses commercial cmdlets (script)

> Description: Information on how to remediate the Chocolatey Package Moderation Rule 0071

Reviewer: ```suggestion
Description: Information on how to remediate the Chocolatey Package Moderation Rule 0071.
```

RuleType: Requirement
---

<?! Include "../../../../../shared/package-validator-rule-requirement.txt" /?>

## Issue

> In an automation script (`.ps1`/`.psm1`), one or more calls to a commercial function that is only available in commercial editions of Chocolatey is being used.

Reviewer: ```suggestion
In an automation script (`.ps1`/`.psm1`), one or more calls to a licensed function that is only available in licensed editions of Chocolatey, is being used.
```

## Recommended Solution

> Please do not use commercial features when creating packages that you will push to the Community repository. Update all automation scripts to use functions that are available in Open Source Chocolatey.

Reviewer: ```suggestion
Please do not use licensed features when creating packages that you will submit to the Chocolatey Community Repository. Update all automation scripts to use functions that are available in open-source Chocolatey.
```

## Reasoning

> Packages created with commercial features should not be made available in Chocolatey Community Repository and are meant to be used by businesses by hosting the package in their internal repository.

Reviewer: ```suggestion
Packages created with licensed features should not be made available in the Chocolatey Community Repository and are intended for use by organizations hosting their own internal package repository.
```

> All packages submitted to the Chocolatey Community Repository are for the community and should work on all editions available for Chocolatey.

Reviewer: ```suggestion
All packages submitted to the Chocolatey Community Repository are for the community and should work on all editions of Chocolatey.
```
